packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,122 @@
+# Calendar
+
+## Component Description
+
+The Calendar lets users view and select a single date or a range of dates. The calendar has five different views: date, month, week, year, and range-picker.
+
+## Design Spec
+
+Due to the incomplete nature of the v9 calendar design spec, work on the component will be based on the calendar from the v8 api while using the skin of the calendar in the v9 date picker design spec.
+
+[Link to v9 Date Picker Design in Figma](https://www.figma.com/file/9EHmS5y1Rr7DCh7KgJBwuz/DatePicker?type=design&mode=design&t=E6sL2ffuuZfwCSRp-0)
+
+[Link to Fluent v8 Calendar Component](https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar)
+
+## Engineering Spec
+
+### Inputs
+
+| attribute                  | type                                                    | default           | description                                                                                                                       |
+| -------------------------- | ------------------------------------------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `type`                     | "date" \| "month" \| "week" \| "year" \| "range-picker" | "date"            | The type of the calendar                                                                                                          |
+| `month-picker-visible`     | boolean                                                 | `true`            | Whether the month picker is shown beside the day picker or hidden                                                                 |
```

Comment:
Another way to use boolean in your code:
Your spec can then show the default as having no value.
You can further reduce component code by adding the attribute to the DOM only when the component needs to be modified.
This works when the default state is no value for the boolean (ie: it's not even present). See below:

<my-component disabled="false">I am a component</my-component>
<my-component disabled="true">I am a component</my-component>
-vs- 
<my-component>I am a component</my-component>
<my-component disabled>I am a component</my-component>

Reply:
Edited the default values for booleans to be empty instead of false!

Reply:
hah, I just noticed how github rendered the code I added to my comment. No wonder you were puzzled.